### ISSUE TITLE
Can't build Docker Image Locally

### ISSUE THREAD

**Issue by Saurav-D** [Unity-Technologies/datasetinsights#112]
**Describe the Bug:**
When trying to build docker image locally, even without any changes, build fails.

**How to Reproduce?**
Get master branch of the repo and try to run `docker build`

**What did you expect to happen:**
Build Successfully.

**Console logs / stack traces**
```
Step 12/14 : RUN poetry config virtualenvs.create false     && poetry install --no-dev
 ---> Running in 8ea17e71488d
Traceback (most recent call last):
  File "/usr/local/bin/poetry", line 7, in <module>
    from poetry.console import main
  File "/usr/local/lib/python3.7/dist-packages/poetry/console/__init__.py", line 1, in <module>
    from .application import Application
  File "/usr/local/lib/python3.7/dist-packages/poetry/console/application.py", line 7, in <module>
    from .commands.about import AboutCommand
  File "/usr/local/lib/python3.7/dist-packages/poetry/console/commands/__init__.py", line 4, in <module>
    from .check import CheckCommand
  File "/usr/local/lib/python3.7/dist-packages/poetry/console/commands/check.py", line 2, in <module>
    from poetry.factory import Factory
  File "/usr/local/lib/python3.7/dist-packages/poetry/factory.py", line 18, in <module>
    from .repositories.pypi_repository import PyPiRepository
  File "/usr/local/lib/python3.7/dist-packages/poetry/repositories/pypi_repository.py", line 33, in <module>
    from ..inspection.info import PackageInfo
  File "/usr/local/lib/python3.7/dist-packages/poetry/inspection/info.py", line 25, in <module>
    from poetry.utils.env import EnvCommandError
  File "/usr/local/lib/python3.7/dist-packages/poetry/utils/env.py", line 22, in <module>
    import virtualenv
  File "/usr/local/lib/python3.7/dist-packages/virtualenv/__init__.py", line 3, in <module>
    from .run import cli_run, session_via_cli
  File "/usr/local/lib/python3.7/dist-packages/virtualenv/run/__init__.py", line 5, in <module>
    from ..app_data import AppDataAction, AppDataDisabled, TempAppData
  File "/usr/local/lib/python3.7/dist-packages/virtualenv/app_data/__init__.py", line 10, in <module>
    from appdirs import user_data_dir
ModuleNotFoundError: No module named 'appdirs'
The command '/bin/sh -c poetry config virtualenvs.create false     && poetry install --no-dev' returned a non-zero code: 1

```

**Environment:**

- OS + version: MacOS
- datasetinsights: latest

**Comment by Saurav-D** [comment]
Running `poetry add appdirs` solves the problem.

**Comment by Saurav-D** [comment]
`standard_init_linux.go:211: exec user process caused "no such file or directory"` I get this error if I add appdirs and use the image in kubeflow.

**Comment by adason** [comment]
Not exactly sure about what's the root cause, but pinning poetry version to `1.0.10` and remove `--no-dev` in poetry install seems to fix the issue.

**Comment by BlairLee** [comment]
It seems we have a new issue. After pinning the poetry version to `1.0.10` and remove `--no-dev` in poetry install, we can successfully build an image and push it to Container Registry. However, when we wanna use our local-built image in Kubeflow, the error comes up at the download stage:
```
standard_init_linux.go:211: exec user process caused "no such file or directory"
```
If you want to reproduce the bug, here is the [run](https://datasetinsights.endpoints.unity-ai-thea-test.cloud.goog/_/pipeline/#/runs/details/88f6d44c-07b2-4132-a5ce-eea1c358b6f6).

**Comment by adason** [comment]
Fixed in #118
